### PR TITLE
PHP-CS-Fixer + StyleCI config

### PR DESCRIPTION
**Pull request by Soullivaneuh**

### CODE REVIEW

File: tools/php-cs/.styleci.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# DO NOT EDIT THIS FILE!
+#
+# It's auto-generated by sonata-project/dev-kit package.
+#
+# Package `sllh/php-cs-fixer-styleci-bridge` is required to get it working.
+
+preset: symfony
+
+enabled:
+  - align_double_arrow
```
Comment (ju1ius) [inline tools/php-cs/.styleci.yml]: I think a lot of people do not like this one...

Comment (Soullivaneuh) [inline tools/php-cs/.styleci.yml]: I understand but it was a @rande request. :stuck_out_tongue: 

For now I'll keep as is to get it working. Feel free to open a PR once this one merged to start a discussion about it. :+1:

Comment (greg0ire) [inline tools/php-cs/.styleci.yml]: See #11

Comment (ju1ius) [inline tools/php-cs/.styleci.yml]: I think the closer it stays from PSR-2 + Symfony cs rules, the less friction for everybody.

Comment (greg0ire) [inline tools/php-cs/.styleci.yml]: @ju1ius discussion is in #11

**Pull request merged**